### PR TITLE
[DF-34] Init project (Nestjs)

### PR DESCRIPTION
**Pull request by pcybxine**

### CODE REVIEW

**Comment by yee2542** [comment]
This setup is not following the acceptance criteria please check.
- `tsconfig.json` enabled `strictNullChecks` , `SynthesisImport` or other to keep a quality of code
- `.prettierc.` check a repo `yee2542/FluentSearch-Storage` to follow it

**Comment by yee2542** [comment]
also set up the GitHub actions to this project

File: package.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
+  "name": "bff",
```
Comment (yee2542) [inline package.json:2]: FluentSearch-BFF

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.eslintrc.js
+.prettierrc
```
Comment (octobrekids) [inline .eslintignore:2]: EOL

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+      - name: Run Tests
+        run: yarn test
+
+      - name: Run E2E Tests
+        run: yarn test:e2e
+
+      - name: Build
+        run: yarn build 
+
```
Comment (octobrekids) [inline .github/workflows/ci.yml:29]: Please add EOL

File: .gitignore [hunk 4]
```diff
@@ -6,99 +10,25 @@ yarn-debug.log*
...
+*.launch
+.settings/
+*.sublime-workspace
+
+# IDE - VSCode
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
```
Comment (octobrekids) [inline .gitignore:34]: EOL

File: .vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
+}
```
Comment (octobrekids) [inline .vscode/settings.json:3]: EOL

**Review by yee2542 — APPROVED** [review]

**Review by octobrekids — COMMENTED** [review]
Please Check

**Pull request merged**